examples/async/async.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates the usage of async functionality in Ballerina, a way to execute operations asynchronously by returning a `future`.
```

Comment:
Suggested edit: description: This sample demonstrates the usage of async functionality in Ballerina - a way to execute operations in an asynchronous manner by returning a `future`.

(Aynschronous has higher SEO value)